tests/integration/ceph_csi_test.go

```diff
@@ -120,8 +114,8 @@ metadata:
    name: ` + csiSCCephFS + `
 provisioner: cephfs.csi.ceph.com
 parameters:
-    monValueFromSecret: "monitors"
-    provisionVolume: "true"
+    clusterID: ` + namespace + `
+    pool: ` + csiPoolCephFS + `
```

Comment:
`pool` is repeated twice here, and as this is cephfs the pool has to be the metadata pool for the CephFS instance, which has the `<fsname>-metadata` typically.

Reply:
@ShyamsundarR my bad copy paste mistake. forgot to change the key  to `fsName` 
updated now

Reply:
No worries, all good now.